bridge/README.md

```diff
@@ -1,3 +1,389 @@
+to handle both directions of the bridge.
+
+## Setup
+
+### Prerequisites
+
+The bridge is written in Python so it must be available.
+
+```sh
+apt-get update
+apt-get install python3 python3-pip libssl-dev
+```
+
+### Creating Virtual Environment (Optional)
+
+```sh
+apt-get install python3-venv
+
+cd symbol-product-directory/bridge
+python3 -m venv venv
+. venv/bin/activate
+```
+
+### Installation
+
+```sh
+cd symbol-product-directory/bridge
+python3 -m pip install -r requirements.txt
+```
+
+## Manual Operation
+
+### Running the Bridge
+
+The following scripts must be executed periodically from `symbol-product-directory/bridge`, in the order shown:
+
+```sh
+. ./venv/bin/activate
+python3 -m workflows.download_balance_changes --config configuration.ini
+python3 -m workflows.download_wrap_requests --config configuration.ini
+python3 -m workflows.download_wrap_requests --config configuration.ini --unwrap
+python3 -m workflows.send_payouts --config configuration.ini
+python3 -m workflows.send_payouts --config configuration.ini --unwrap
+python3 -m workflows.check_finalized_transactions --config configuration.ini
+python3 -m workflows.check_finalized_transactions --config configuration.ini --unwrap
+```
+
+If the bridge is configured in **native mode** (exchanging native XYM for native ETH),
+all commands with the `--unwrap` parameter must be skipped.
+In this mode, the bridge operates only in one direction.
+
+All commands require a configuration file, specified with the `--config` parameter.
+The expected contents of this file are described in the [Configuration](#configuration) section.
+
+The more frequently the scripts are run, the faster bridging requests are processed,
+but at the cost of higher hardware usage.
+Considering the average block times of the supported networks, running the scripts every 15 seconds provides
+a good balance.
+
+### Exposing the API
+
+The bridge exposes an HTTP API which allows external systems to query its status.
+For example, the API can return the list of pending wrap and unwrap requests for a given account,
+or provide details about past errors.
+An OpenAPI specification for this API will be provided soon.
+
+The API has been developed using [Flask](https://flask.palletsprojects.com/en/stable/).
+To expose it, run:
+
+```sh
+. ../venv/bin/activate
+export FLASK_RUN_PORT=5000
+export BRIDGE_API_SETTINGS=/path/to/api_configuration.ini
+FLASK_APP=bridge.api:create_app FLASK_ENV=development PYTHONPATH=../. python3 -m flask run
+```
+
+The API requires its own configuration file (`api_configuration.ini` in the command above):
+
+```ini
+CONFIG_PATH="<path_to_bridge_configuration.ini>"
+```
+
+## Running using Docker
+
+As an alternative to manually installing dependencies and running scripts, you can use Docker.
+
+### Running the Bridge
+
+```sh
+docker run -d -it --name xym_wxym_bridge --restart always -v $(pwd):$(pwd) symbolplatform/bridge:1.0 /home/ubuntu/product/bridge/xym_wxym_bridge/run-bridge-docker.sh
```

Comment:
I made some changes to the Dockerfile that is now checked in.  
To run the Docker image - 
``docker run -d -it --name xym_wxym_bridge --restart always -v $(pwd):/data symbolplatform/bridge:1.0 api <mode>``

Mode will be one of these
api - to start the API (also add the ``-p 5000:5000`` to expose the api port)
nativeflow - to run the native flow
wrappedflow - to run the wrapped flow

 By default, the bridge configuration file would be in ``/data/config/configuration.ini``
 This can be overridden by setting the ``BRIDGE_CONFIG_PATH`` environment variable

``docker run -d -it --name xym_wxym_bridge --restart always -v $(pwd):/data -e BRIDGE_CONFIG_PATH=<path_to_config> symbolplatform/bridge:1.0 api <mode>``

Reply:
Thank you. I will do one more PR with the parameters that have been changed lately. So I will add this change too. For this PR, I would just merge as is now.